### PR TITLE
Refactored Edb documentation

### PR DESCRIPTION
**Pull request by maxcapodi78**

### CODE REVIEW

File: doc/source/EDBAPI/NetsEdb.rst [hunk 1]
```diff
@@ -0,0 +1,49 @@
+Components
+==========
+This sections contains API references for net management.
```
Comment (gmalinve) [inline doc/source/EDBAPI/NetsEdb.rst:3]: @maxcapodi78 section :)

File: doc/source/EDBAPI/PadstackEdb.rst [hunk 2]
```diff
@@ -1,20 +1,35 @@
-EDB padstack
-============
-These classes are the containers of data management for padstacks and padstack definitions.
+vias and padstacks
+==================
+This sections contains API references for padstack management.
```
Comment (gmalinve) [inline doc/source/EDBAPI/PadstackEdb.rst:3]: @maxcapodi78 section or these if it is plural :)

Comment (gmalinve) [reply doc/source/EDBAPI/PadstackEdb.rst:3]: for both comments change verb "contain" accordingly

**Review by gmalinve — APPROVED** [review]

File: doc/source/EDBAPI/NetsEdb.rst [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+
+
+.. currentmodule:: pyaedt.edb_core.edb_data.nets_data
+
+.. autosummary::
+   :toctree: _autosummary
+   :nosignatures:
+
+
```
Comment (MaxJPRey) [inline doc/source/EDBAPI/NetsEdb.rst:37]:
```suggestion
```

File: doc/source/EDBAPI/PrimitivesEdb.rst [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+These classes are the containers of data management for primitives and arcs.
+
+.. currentmodule:: pyaedt.edb_core.edb_data.primitives_data
+
+.. autosummary::
+   :toctree: _autosummary
+   :nosignatures:
+
+
+
```
Comment (MaxJPRey) [inline doc/source/EDBAPI/PrimitivesEdb.rst:31]:
```suggestion
```

**Review by Alberto-DM — APPROVED** [review]
LGTM. Change from version 2022.2 to 2023.1

File: doc/source/EDBAPI/NetsEdb.rst (hunk 3 above)
Comment (maxcapodi78) [reply doc/source/EDBAPI/NetsEdb.rst]: Done

File: doc/source/EDBAPI/NetsEdb.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (maxcapodi78) [reply doc/source/EDBAPI/NetsEdb.rst]: Done. Thanks

**Pull request merged**